nix/module.nix

```diff
@@ -306,6 +307,12 @@ in
       description = "As semaphore, the max amount of parallel get sync record requests to perform.";
     };
 
+    maxVDFValidationThreadCount = mkOption {
+      type = types.int;
+      default = 10;
```

Comment:
FWIW the default [used in ar.erl](https://github.com/ArweaveTeam/arweave/blob/master/apps/arweave/include/ar_config.hrl#L88-L89) if the `max_vdf_validation_thread_count` is not provided is `schedulers_online / 2` - which translates roughly to half the available cores.

Reply:
Not sure if this is relevant to your use case, though!

Reply:
That's true, we may want to add that as a comment for users to consider. But given the dynamic nature of "schedulers_online" it's not really easy or even possible to infer that from a static configuration. I'll add it as a comment!

Reply:
Well I just realized, that I can make it default to null to not interfere with the default dynamic behavior. Only when user passes a value it will be set, otherwise not. Then it's the same behavior.

Reply:
Agree - comment works for me.

Unless there's an easy way to not include `max_vdf_validation_thread_count` in the generated config.json if it isn't provided? That would have the node use the `ar.erl` hardcoded default.

Reply:
yes it was easy to allow for nullability (kudos to chatgpt for the null filter function), I'm testing this now on a node. I can give heads up when I've confirmed that this works as intended.

Reply:
hahah, I used chatgpt to explain the filter function to me! Oh man, I'm seeing a future where ChatGPT trolls us all by returning some gibberish and then pretending like the gibberish does exactly what it's supposed to.

Reply:
turned out chatgpt did some blunders, the function is fixed now as well as the nullability, which required the type `type = with types; nullOr int;`